NYCDropInCenters/nyc_drop_in_centers.py

```diff
@@ -0,0 +1,66 @@
+import os
+import sys
+from datetime import datetime
+import re
+import numpy as np
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+from pymongo import MongoClient, errors
+from tqdm import tqdm
+
+_i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _i not in sys.path:
+    # add parent directory to sys.path so utils module is accessible
+    sys.path.insert(0, _i)
+del _i  # clean up global name space
+from shared_code.utils import (
+    check_similarity, locate_potential_duplicate,
+    insert_services, client
+)
+from shared_code.base_scraper import BaseScraper
+
+
+class NYC_Scraper(BaseScraper):
+
+    def grab_data(self):
+
+        df = pd.read_csv(self.data_url, usecols=self.extract_usecols)
+        df.drop_duplicates(subset=self.drop_duplicates_columns,
+                           inplace=True,
+                           ignore_index=True
+                           )
+        df.rename(columns=self.rename_columns, inplace=True)
+        df[['address1']] = df['address1'].str.replace('\n', ' ')
+        df[['address1']] = df['address1'].str.extract('(.+)(?=NY)')
+        df['state'], df['city'], df['serviceSummary'], df['source'] = 'NY', 'New York', self.service_summary, self.source
+        return df
+
+
+nyc_scraper = NYC_Scraper(
+    source = "NYCDropInCenters",
```

Comment:
`source` value and `data_source_collection_name` value should be : nyc_drop_in_centers which will allow us to have join query in future.